Firestore/core/src/firebase/firestore/remote/remote_event.h

```diff
@@ -74,6 +74,9 @@ class TargetChange {
  public:
   TargetChange() = default;
 
+  explicit TargetChange(bool current) : current_{current} {
```

Comment:
The Android version has this as a static factory named `createSynthesizedTargetChangeForCurrentChange`. This is considerably more explicit and self-documenting than just a constructor taking a boolean. Please port.

Reply:
Done.